README.md

```diff
@@ -46,15 +72,33 @@ Not implemented, but planned.
 
 |Switch                            | Default | Description |
 |----------------------------------|---------|-------------|
-|extractSamplesFromOutputFiles     | false   | If this is true and samples are neither passed by MDT, configuration or sample list, samples are extracted from files in the alignment folder.            |
+|extractSamplesFromOutputFiles     | false   | If this is true and samples are neither passed by metadata table, configuration or sample list, samples are extracted from files in the alignment folder.            |
 |extractSampleNameOnlyFromBamFiles | false   | By default, the method will search for samples in all files in the alignment directory. With this switch, you can restrict it to BAM files.            |
 
 ## Changelist
 
-    - Add new method for sample from filename extraction
-
-...
-
+  * Version update to 1.3.0
+    - Change tests to extend the RoddyTestSpec class in Roddy 3.4.0
+    - Changed tests in WorkflowUsingMergedBamsSpec to use flags instead of getContext() method.
+      * The ContextResource @Rule did not work anymore for the tests.
+      * Makes it easier to read, flags are boolean values now.
+      * Context and file objects are now created in the test methods.
+    - Updated copyright notes.
+    - Added compareTo(), equals() and hashCode() to Sample class.
+    - In COMetadataAccessor class:
+      * Added the 'extractSampleNameOnlyFromBamFiles' flag which tells extractSamplesFromOutputFiles() to ignore non BAM files.
```

Comment:
Not in COMetadataAccessor: Use an appropriate subclass. The idea of the COMetadataAccessor interface was to have subclass for directory/paths, table, XML, JSON, YAML, database, ... . The right subclass would be instantiated as strategy at another place, that (1) decides based on the parameters which strategy to use and (2) parametrizes the subclass with appropriate options.

Reply:
Ad discussed, we'll rework the whole model in a later version. Skip this for now.